strings/src/main/res/values/strings.xml

```diff
@@ -1132,18 +1132,23 @@
     # Entities
     ##############################################
 
-    <!-- An entity is any person, place or thing that a form can be about. This title appears on an experimental settings screen to view local entities managed in the current project. -->
-    <string name="entities_title">Local entities</string>
+    <!-- An entity is any person, place or thing that a form can be about. This title appears on an experimental settings screen to view entities managed in the current project. -->
+    <string name="entities_title">Entities</string>
 
-    <!-- Title for setting that controls whether locally created/updated entities appear in forms -->
-    <string name="include_local_entities_setting">Include local entities in forms</string>
+    <!-- Title for setting that controls whether local entity features are enabled or not -->
+    <string name="include_local_entities_setting">Enable local entities</string>
 
-    <!-- Text for button that leads to browser for the user to view and manage local entites -->
-    <string name="entity_browser_button">View local entities</string>
+    <string name="local_entity_features_summary">Follow up forms will have consistent entity lists and will include locally created or updated entities.</string>
+
+    <!-- Text for button that leads to browser for the user to view and manage entity lists -->
+    <string name="entity_browser_button">View entity lists</string>
 
     <!-- Text for button that deletes all entities for the project -->
     <string name="clear_entities">Clear all</string>
 
+    <!-- Text for button that adds a new entity list -->
+    <string name="add_entity_list">Add entity list</string>
```

Comment:
Entity list is the same as dataset right? Here the naming is also confusing.

Reply:
That was a change in terminology that's happened since we added the first entity code to Collect. The [spec now states](https://getodk.github.io/xforms-spec/entities.html#entities):

> In this specification, collections of Entities are referred to as Datasets. The term “Entity List” is generally recommended instead of “Dataset” in text that is intended for users rather than developers.

We could just call them "entity lists" everywhere in Collect and avoid "datasets" entirely?

Reply:
Yeah, it would be better to use consistent naming.

Reply:
Again, I think we should consolidate this in #6011. Agreed that we want to use "entity list" everywhere instead of data sets though.